src/aggregator/aggregator/options.go

```diff
@@ -328,6 +335,7 @@ type options struct {
 	bufferDurationAfterShardCutoff   time.Duration
 	flushManager                     FlushManager
 	flushHandler                     handler.Handler
+	passthroughWriter                writer.Writer
```

Comment:
mind setting a default value in the NewOptions() ctor for this field as well. can blackhole it, just want the to avoid non usable default values.